### ISSUE TITLE
Ecuador phone number

### ISSUE THREAD

**Issue by JoynerPerez1987** [googlei18n/libphonenumber#1987]
In Ecuador the cell phone numbers begin with:
095, 096, 097, 098, 099
Example:
+593 95 ...
+593 96 ...
...
+593 99 ...
Please help me

**Comment by penmetsaa** [comment]
@JoynerPerez1987,
Thank you for taking the time to improve libphonenumber!

We already support specific phone number ranges that starts with 95-99 as mobile. Ex: 958X XXXXX. Do you say that all 9 digit 9[5-9]XX XXXXX phone numbers are valid Ecuador mobile numbers?

Please share any working numbers or evidence you may have. See our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).

**Comment by kkeshava** [comment]
This issue is blocked on your or other GitHub contributors' input. Please help us find evidence that all 9 digit 9[5-9]XX XXXXX is valid, or provide an example number from a business website or similar. Otherwise we don't have enough information to fix the issue and may have to close it as infeasible.

**Comment by penmetsaa** [comment]
I'm closing as this issue is infeasible for us to fix at this point (we need more information). Please re-open if you respond so we pick up on it.

**Comment by JoynerPerez1987** [comment]
Sorry. From 95 to 99 if it works, but I need to recognize
+593 94 ...
+593 93 ...

**Comment by penmetsaa** [comment]
Same is the case with 93 and 94 mobile ranges. We already support valid subranges under 93 and 94. Please check results at our [demo](https://libphonenumber.appspot.com/). 
Ex: https://libphonenumber.appspot.com/phonenumberparser?number=%2B593+93+923+8783

If any valid number is marked as invalid in our demo portal, please create a new issue with supporting evidences. See our [guidelines](https://github.com/googlei18n/libphonenumber/blob/master/CONTRIBUTING.md#provide-supporting-evidence).
